profiler/profile_test.go

```diff
@@ -7,20 +7,32 @@ package profiler
 
 import (
 	"bytes"
+	"context"
+	"encoding/json"
 	"fmt"
 	"io"
 	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
 	"os"
+	"runtime/trace"
 	"strconv"
 	"strings"
+	"sync"
 	"testing"
 	"time"
 
+	httptrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/julienschmidt/httprouter"
+
 	"gopkg.in/DataDog/dd-trace-go.v1/profiler/internal/pprofutils"
 
 	pprofile "github.com/google/pprof/profile"
+	"github.com/julienschmidt/httprouter"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
```

Comment:
Small issue grouping here. We should have:
1. Standard library
2. Local (dd-trace-go) imports
3. Other stuff
Separated by a blank line.